### DISCUSSION TITLE
Eufy indoor Cam RTSP streaming works intermittently in HomeKit

### DISCUSSION THREAD

**Discussion by wbizreh** [koush/scrypted#977] [Q&A]
Eufy indoor Cam RTSP streaming works intermittently in HomeKit. Sometimes I get a no response and sometime it just take forever to start live streaming in other instances it works perfectly. Can’t pin point the issue.  I am using scrypted RTSP plug In on a Linux machine.

**Comment by Magna66** [comment]
Had the same problem with Eufy indoor, go the camera, then stream, stream stream1 in RTSP parser select FFmpeg (TCP). Worked for me no more stuttering.
